### ISSUE TITLE
Kustomize Version in 1.17

### ISSUE THREAD

**Issue by strangiato** [kubernetes-sigs/kustomize#3075]
Which version of Kustomize ships with K8s 1.17?

The main readme lists the kustomize version numbers that ship with 1.14-1.16 but does not list version 1.17+.  It would be great to get the readme updated with this information.

**Comment by Shell32-Natsu** [comment]
I guess we haven't updated the kustomize version that shipped with kubectl for a long time. There are some technical reasons which forbid us to use latest kustomize in kubectl. We are working on re-integrating kustomize to kubectl.

**Comment by strangiato** [comment]
Yeah I did see that upgrading was being worked on #1500 but I wasn't sure if that meant it was still running 2.0.3.

I'm trying to determine if an issue that appears to have been resolved in 3.0.0 (#1082) is because kubectl ships with the older version or if it is an issue on my end.  I'm assuming the version running in 1.17 is still 2.0.3.

It would be great if we had a way to check the kustomize version moving forward.

Something like this would be very helpful:

```sh
kubectl kustomize --version
```
